crates/iota-indexer/src/indexer_reader.rs

```diff
@@ -1579,22 +1579,34 @@ impl IndexerReader {
     }
 
     pub fn get_latest_move_call_metrics(&self) -> IndexerResult<MoveCallMetrics> {
+        let latest_epoch = run_query!(&self.pool, |conn| {
+            move_call_metrics::table
+                .select(move_call_metrics::dsl::epoch)
+                .order(move_call_metrics::dsl::epoch.desc())
+                .limit(1)
+                .first::<i64>(conn)
+        })
+        .map_err(|_| IndexerError::PostgresRead("record not found".to_string()))?;
```

Comment:
It's OK to have a generic error message or `latest epoch record not found` could suit better as error message?

Reply:
Yes, better me more clear about it!